background/rust.tex

```diff
@@ -8,6 +8,23 @@ \section{The Rust Programming Language} % (fold)
 The language borrows many constructs and features from other programming languages, some of which are described in the following sections.
 Over the course of {\rust}'s development, the language has set out to solve two major problems concerning both safety and concurrency, as well as taking full utilization of the underlying hardware.
 
+\subsection{Hello World}
```

Comment:
Hva med å fjerne "Hello World" i implementation når det legges til i background?

Blir vel litt rart å vise to "Hello World" i løpet av rapporten?